src/styles/tools/ec-dropdown.css

```diff
@@ -0,0 +1,13 @@
+@define-mixin ec-dropdown-search-item
+  $item-horizontal-padding: 8px,
+  $item-vertical-padding: 16px {
+  padding: $item-horizontal-padding $item-vertical-padding;
+}
+
+@define-mixin ec-dropdown-search-item-hover-effect $background-color, $color {
+  &:hover {
+    cursor: pointer;
```

Comment:
Could you check if we can use TW here?

Reply:
- [ ] Use TW

Reply:
@mcibique we cannot use `@apply` it is not picked by the plugin

Reply:
Are you sure, I see other mixins doing that, e.g.:
```@define-mixin ec-alert-responsive {
  @apply tw-flex-row;
  @apply tw-py-16 tw-px-24;

  .ec-alert__icon {
    @apply tw-w-24 tw-h-24;
    @apply tw-my-4 tw-mr-16 tw-ml-0;

    &--alert-has-subtitle {
      @apply tw-self-start;
    }
  }

  .ec-alert__content {
    @apply tw-text-left;
  }

  .ec-alert__button {
    @apply tw-my-0 tw-mx-24;
  }

  @mixin-content;
}
```

Reply:
Weird, I'll double check thanks